### ISSUE TITLE
Anzeige des Veranstaltungsortes auf einer Karte

### ISSUE THREAD

**Issue by ghost** [CodeforChemnitz/TheaterWecker#13]
Nach Benachrichtigung über verfügbare Tickets, wäre es sinnvoll auf einer Karte anzuzeigen, wo die Veranstaltung stattfindet.

**Comment by ronnyhartenstein** [comment]
Wir sind uns nach einer internen Diskussion noch uneins, ob die Mehrzahl der Nutzerschaft nicht ohnehin die üblichen paar Kulturorte in Chemnitz kennt...

**Comment by ghost** [comment]
Eure Nutzer kenne ich natürlich nicht so gut wie ihr. Hätte nur vermutet, dass viele Studenten nicht schon immer in Chemnitz wohnen, sondern für das Studium erst dort hinziehen.

**Comment by ronnyhartenstein** [comment]
Klar, wird noch kommen, erstmal die Location im Backend sauber herausqualifizieren (so viele sind das in C ja nicht) und dann im Frontend und App hübsch darstellen.

**Comment by benediktg** [comment]
@ronnyhartenstein Das hier könntest du doch dafür nehmen, oder? https://github.com/airbnb/react-native-maps

**Comment by ronnyhartenstein** [comment]
@benediktg Logisch 👍
